common/scripts/changelog/check.mjs

```diff
@@ -16,15 +16,19 @@ import { CHANGE_DIR_BETA, CHANGE_DIR_STABLE } from "./constants.mjs";
 
 async function main() {
   const [base, head] = parseArgs(process.argv);
-  const gitLogStdoutStableChangeFiles = await exec_output(`git log --name-status ${base}..${head} -- ${CHANGE_DIR_STABLE}`);
-  const gitLogStdoutBetaChangeFiles = await exec_output(`git log --name-status ${base}..${head} -- ${CHANGE_DIR_BETA}`);
 
-  const newStableChangeFiles = parseNewChangeFiles(gitLogStdoutStableChangeFiles);
-  const newBetaChangeFiles = parseNewChangeFiles(gitLogStdoutBetaChangeFiles);
-  const newChangeFilesCount = (newStableChangeFiles?.length ?? 0) + (newBetaChangeFiles?.length ?? 0);
+  // Check if the changelog files are present
+  const changedStableFiles = await exec_output(`git diff --diff-filter=A --name-status ${base}..${head} -- ${CHANGE_DIR_STABLE}`);
```

Comment:
Can you add a comment here why `A` indicates change files, I am still unclear how this equates to a changefile never mind someone else in many months time

Reply:
Added the comments and a link to git documentation